pages/api/v1/listing/user-posts.ts

```
@@ -166,281 +167,284 @@ export const getUserPosts: TGetUserPosts = async (params) => {
+			subsquidRes = await fetchSubsquid({
+				network,
+				query: GET_ONCHAIN_POSTS_BY_PROPOSER_ADDRESSES,
+				variables: {
+					proposer_in: (addresses as string[])?.map((address) => getEncodedAddress(address, network)) || []
+				}
+			});
+			const edges = subsquidRes?.data?.proposalsConnection?.edges;
+			if (edges && Array.isArray(edges)) {
+				const onChainPostsPromise = edges?.map(async (edge) => {
+					if (edge && edge.node) {
+						const { type, hash, index, createdAt, description, proposalArguments, proposer, preimage, trackNumber, tally, status: resultStatus, statusHistory, group } = edge.node;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)